.github/workflows/build.yml

```diff
@@ -0,0 +1,53 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: Test
+
+on:
+  push:
+    branches:
+      - '*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        container: [ "python:3.8", "python:3.9", "python:3.10" ]
```

Comment:
Interesting... I didn't realise you could do this. Is there any benefit to doing it this way rather than using the default ubuntu image and setting up the python version using the relevant action. 

Like this: https://github.com/Flagsmith/flagsmith/blob/main/.github/workflows/api-pull-request.yml#L41